glossary.md

```diff
@@ -16,6 +16,15 @@ For example, namespaces, resource limits, and mounts are all part of the contain
 ## Container namespace
 
 On Linux, a leaf in the [namespace][namespaces.7] hierarchy in which the [configured process](config.md#process-configuration) executes.
+For example, with a PID namespace hierarchy like:
+
+| PID namespace | Parent namespace | Runtime process PID | Container process PID |
+| ------------- | -----------------| ------------------- | --------------------- |
+| *A*           | -                | 123                 | 678                   |
+| *B*           | *A*              | 45                  | 90                    |
+| *C*           | *B*              | 0                   | 1                     |
+
+where the container process was launched in a new [PID namespace][pid_namespaces.7], namespace *C* (in which the container process is executing) would be the container PID namespace.
 
```

Comment:
im not sure if we want to have nested user namespace clarification / education here. Seems much better to link to the kernel pages instead http://man7.org/linux/man-pages/man7/user_namespaces.7.html